midp/gfx.js

```diff
@@ -396,10 +396,6 @@ var currentlyFocusedTextEditor;
         return defaultFont;
     }
 
-    Override["javax/microedition/lcdui/Font.getDefaultFont.()Ljavax/microedition/lcdui/Font;"] = function() {
```

Comment:
If we remove `getDefaultFont` override,  we end up with two `getDefaultFont()`. One in this file gfx.js and one in Font.java. They will create two default font instance, but actually we only need one of them.